Dfe.Identifiers.Api/Interfaces/IEstablishmentRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Dfe.Identifiers.Api.Models;
+
+namespace Dfe.Identifiers.Api.Interfaces;
+
+public interface IEstablishmentRepository
```

Comment:
Is this a repository? It can only Get by a specific param, maybe a better name would be Queries